src/test/java/com/hotels/bdp/circustrain/bigquery/table/service/unpartitioned/UnpartitionedTableServiceTest.java

```diff
@@ -0,0 +1,51 @@
+/**
+ * Copyright (C) 2018 Expedia Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.hotels.bdp.circustrain.bigquery.table.service.unpartitioned;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+import org.apache.hadoop.hive.metastore.api.Table;
+import org.junit.Before;
+import org.junit.Test;
+
+import com.hotels.bdp.circustrain.bigquery.conversion.BigQueryToHiveTableConverter;
+
+public class UnpartitionedTableServiceTest {
+
+  private UnpartitionedTableService service;
+  private Table table = new Table();
+
+  @Before
+  public void init() {
+    table = new BigQueryToHiveTableConverter().withTableName("foo").withDatabaseName("bar").convert();
```

Comment:
Can't all of this be done inline above and then you won't need an `init()` method?